### ISSUE TITLE
Bug: Landscape tablet layout

### ISSUE THREAD

**Issue by ghost** [aforty/sabnzbd-knockstrap#5]
Minor layout bug with Tablet (e.g. Apple iPad 2-3rd, mini).
To simulate this I used Viewport Resizer (http://lab.maltewassermann.com/viewport-resizer/).

Size: 768x1024 (Portrait), Ratio: 3:4:
![6j0csvt](https://f.cloud.github.com/assets/1141690/724607/a9556a0c-e039-11e2-8511-e0ce8b05666b.png)

Size: 1024x768 (Landscape), Ratio: 4:3
![slen75f](https://f.cloud.github.com/assets/1141690/724605/a5b03558-e039-11e2-9445-4726348f7f36.png)

**Comment by aforty** [comment]
Nice download speed.

Keep it coming... :+1:

**Comment by ghost** [comment]
Ooh... wait... it's a language issue. English is fine, but not in Dutch.

**Comment by aforty** [comment]
Bigger issue is probably that the theme is not localized in any way. Tough to do because I use verbiage that differs from the translations SAB provides.

**Comment by ghost** [comment]
Yeah... Shorten words if they are to long and when hovering complete the words?
